### PR TITLE
chore: prepare release

### PR DESCRIPTION
**Pull request by AndreasZeissner**
[Uploading Kooha-2024-09-16-17-03-09.webm…]()

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,13 +28,146 @@ The Cosmo Terraform provider includes the following resources and data sources:
...
 
 ## Usage
 
-To use the Cosmo Terraform provider:
+Ensure to set `COSMO_API_URL` and `COSMO_API_KEY` environment variables to point to your cosmo setup.
+
+For example:
+
+```bash
+export COSMO_API_KEY="cosmo_669b576aaadc10ee1ae81d9193425705"
```
Comment (StarpTech) [inline README.md]: Let's not insert a real token here even if it is deactivated. Use a placeholder.

File: docs/resources/subgraph.md [hunk 2]
```diff
@@ -36,16 +36,16 @@ resource "cosmo_subgraph" "test" {
...
-- `namespace` (String) The namespace in which the subgraph is located.
 - `routing_url` (String) The routing URL of the subgraph.
 
 ### Optional
 
-- `headers` (List of String) Headers for the subgraph.
 - `is_event_driven_graph` (Boolean) Indicates if the subgraph is event-driven.
 - `is_feature_subgraph` (Boolean) Indicates if the subgraph is a feature subgraph.
 - `labels` (Map of String) Labels for the subgraph.
+- `namespace` (String) The namespace in which the subgraph is located.
```
Comment (StarpTech) [inline docs/resources/subgraph.md:46]: Is this auto generated?

**Review by StarpTech — CHANGES_REQUESTED** [review]

File: docs/resources/subgraph.md (hunk 2 above)
Comment (AndreasZeissner) [reply docs/resources/subgraph.md:46]: Yes it comes from the schemas: https://github.com/wundergraph/terraform-provider-cosmo/blob/fabbe314d0c1e5679d0fa0ba080684683ae8ee53/internal/service/federated-graph/resource_cosmo_federated_graph.go#L58

**Review by StarpTech — APPROVED** [review]
LGTM

**Pull request merged**